iis-wf/iis-wf-primary/src/main/resources/eu/dnetlib/iis/wf/primary/processing/oozie_app/workflow.xml

```diff
@@ -382,6 +382,14 @@
                     <name>output_document_to_research_initiatives</name>
                     <value>${output_document_to_project_concepts}</value>
                 </property>
+                <property>
+                    <name>output_report_relative_path</name>
```

Comment:
Again, conventions: we should stick to only one of them. You are using camelCase, up until now phrases were separated with underscore. This is also the main separator used in all hdfs directories generated by IIS.